packages/neso/package.py

```diff
@@ -49,17 +49,19 @@ class Neso(CMakePackage):
     variant(
         "coverage", default=False, description="Enable coverage reporting for GCC/Clang"
     )
+    variant(
+        "nvcxx", default=False, description="Enable compilation using nvcxx"
+    )
 
     # Some SYCL packages require a specific run-time environment to be set
     depends_on("sycl", type=("build", "link"))
     depends_on("intel-oneapi-dpl", when="^dpcpp", type="link")
     depends_on("fftw-api", type="link")
-    depends_on("nektar@5.2.0-2022-09-03+compflow_solver", type="link")
```

Comment:
Is there a reason you removed this version requirement?

Reply:
Same thing as in the comment for the `depends_on("mpi", ...)` deletion

Reply:
This should be added back now (see message for commit e2b6e79)